apps/web/src/components/Composer/Actions/Attachment.tsx

```diff
@@ -173,7 +154,7 @@ const Attachment: FC = () => {
               accept={ALLOWED_AUDIO_TYPES.join(',')}
               className="hidden"
               onChange={handleAttachment}
-              disabled={attachments.length >= 4}
+              disabled={!!attachments.length}
```

Comment:
Typecast this to `Boolean(attachments.length)` 🙇🏼

Reply:
Done